icu4c/source/data/misc/supplementalData.txt

```diff
@@ -11357,6 +11362,7 @@ supplementalData:table(nofallback){
                 "energy-british-thermal-unit",
                 "force-pound-force",
                 "force-newton",
+                "force-kilowatt-hour-per-100-kilometer",
```

Comment:
Huh? That's a measure of consumption, not of force.

Reply:
By definition, energy over distance equals force.  So the "real" category should be force.  But the "fake" category could still be consuption if that makes more sense.

Reply:
kWh/100km should go together with
```
                "consumption-liter-per-100-kilometer",
                "consumption-liter-per-kilometer",
                "consumption-mile-per-gallon",
                "consumption-mile-per-gallon-imperial",
```

Reply:
https://unicode-org.atlassian.net/browse/CLDR-14837